orchestration/safe_agent_monitor.py

```
@@ -0,0 +1,164 @@
+            result = subprocess.run(
+                ["tmux", "ls"], check=False, capture_output=True, text=True
+            )
+            if result.returncode == 0:
+                sessions = []
+                for line in result.stdout.strip().split("\n"):
+                    if "agent" in line:
+                        session_name = line.split(":")[0]
+                        sessions.append(session_name)
+                return sessions
+            return []
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)